mcs_benchmark_data/models/benchmark_choice.py

```diff
@@ -15,7 +15,15 @@ class BenchmarkChoice(_Model):
     '''A possible choice for a benchmark sample'''

```

Comment:
A triple-quoted string can have multiple lines in it. Docstrings should be a single triple-quoted string.